doc/releases/compatibility.adoc

```diff
@@ -0,0 +1,11 @@
+= Compatibility
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Et malesuada fames ac turpis egestas.
+Mauris pharetra et ultrices neque ornare aenean euismod elementum nisi.
+Nulla facilisi morbi tempus iaculis urna id. Justo eget magna fermentum iaculis.
+Sed augue lacus viverra vitae congue eu.
+Velit ut tortor pretium viverra suspendisse.
+Amet commodo nulla facilisi nullam vehicula ipsum a.
+Nibh nisl condimentum id venenatis a.
+Diam vel quam elementum pulvinar etiam non quam lacus suspendisse.
```

Comment:
@max-rosin Same here. Even if there is no content yet, maybe a hint to contact the ASAM office?

Reply:
@max-rosin opinion?

Reply:
@stefancyliax: [The following topic from the existing documentation](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/description.html#compatibility) holds the content that was originally intended to go into here. The original topic defines the terms "faithful", "forward compatibility" and "backward compatibility". I did not see these terms being used in the documentation. Do we need these definitions? If not, I would opt for removing the topic altogether

On a second note, do we need a dedicated "contact" topic? Or should the contact information be part of the page layout, for example, the footer (as far as I can see, it isn't today)

Reply:
Added an issue for that: https://github.com/OpenSimulationInterface/osi-documentation/issues/39